enterprise/views.py

```diff
@@ -423,6 +424,37 @@ def post_course_specific_consent(self, request, course_id, consent_provided):
             )
         if not consent_provided:
             failure_url = request.POST.get('failure_url') or reverse('dashboard')
+            enterprise_customer_name = request.POST.get('enterprise_customer_name')
+
+            # Check that if we need to add consent decline message from the
+            # provided querystring param `show_consent_decline_notification`.
+            scheme, netloc, path, query_string, fragment = urlsplit(failure_url)
+            url_params = parse_qs(query_string)
+            # pylint: disable=invalid-name
+            show_consent_decline_notification = url_params.pop('show_consent_decline_notification', None)
+
+            if show_consent_decline_notification:
+                messages.warning(
+                    request,
+                    _(
+                        '{span_start}We could not enroll you in {italic_start}{course_name}{italic_end}.{span_end} '
```

Comment:
these are placeholders for html tags and should not affect translation strings. I have added `.po` files.